### PR TITLE
Add webapp-module-modal

### PR DESCRIPTION
**Pull request by lipis**
https://github.com/wireapp/webapp-module-modal

### CODE REVIEW

File: packages/modal/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "grunt-contrib-copy": "1.0.0",
+    "grunt-contrib-jasmine": "1.1.0",
+    "grunt-contrib-less": "1.4.1",
+    "grunt-contrib-watch": "1.0.0",
+    "grunt-open": "0.2.3",
+    "load-grunt-tasks": "3.5.2"
+  },
+  "files": ["app/script/", "app/style/", "dist/script/", "dist/style/"],
+  "license": "GPL-3.0",
+  "name": "@wireapp/modal",
```
Comment (bennyn) [inline packages/modal/package.json]: After talking to @AndyTheGiant & @Yserz we should brand that module with the name `@wireapp/jquery-modal`.

File: packages/modal/bower.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "webapp-module-namespace": "https://github.com/wireapp/webapp-module-namespace.git#1.0.0"
+  },
+  "devDependencies": {
+    "jquery": "~2",
+    "lesshat": "~3",
+    "keyboardjs": "https://github.com/RobertWHurst/KeyboardJS.git#2.0.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wireapp/webapp-module-modal"
```
Comment (bennyn) [inline packages/modal/bower.json]: URL needs to be adjusted:

```json
"repository": "https://github.com/wireapp/wire-web-packages/tree/master/packages/jquery-modal",
```

File: packages/modal/app/style/webapp-module-modal.less [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    // close
+    .modal-content-anim-close();
+    display: block;
+    background-color: @bg-color;
+    border-radius: 4px;
+    overflow: hidden;
+  }
+}
+
+// mixings
```
Comment (bennyn) [inline packages/modal/app/style/webapp-module-modal.less]: These things are called http://lesscss.org/features/#mixins-feature

Comment (lipis) [reply packages/modal/app/style/webapp-module-modal.less]: hehe

File: packages/modal/package.json (hunk 1 above)
Comment (lipis) [reply packages/modal/package.json]: agree

**Comment by lipis** [comment]
I also renamed the filenames.. so whenever we are going to use this we should load the `jquery-modal.js`

**Comment by lipis** [comment]
You have to show me how to publish new packages from here.. do we merge first? how it's done..?

**Pull request closed without merging**